src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static Todo[] list = new Todo[100];
+    public static int index = 0;
+
+    public static void addToList(Todo task) {
+        list[index] = task;
+        index++;
+    }
+
+    public static void printList() {
+        for (int i = 0; i < index; i++) {
+            System.out.println(String.valueOf(i + 1) + ". " + list[i].toString());
+        }
+    }
+
+    public static void printAddMessage(Todo task) {
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + task.toString());
+        System.out.println("Now you have " + String.valueOf(index) + " tasks in the list.");
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
```

Comment:
Perhaps you should not have added 4 more indentations here as initially, it was 8 indentation which fulfills the coding standing